plugins/auth-auth0/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@amplication/plugin-auth-auth0",
+  "version": "1.0.0",
+  "description": "Auth0 plugin for Amplication",
+  "main": "dist/index.js",
+  "scripts": {
+    "prepublishOnly": "npm run build",
+    "dev": "webpack --watch",
+    "build": "webpack",
+    "prebuild": "rimraf dist",
+    "format:write": "prettier --write \"**/*.{js,ts}\"",
+    "lint:fix": "eslint './src/**/*.{ts, tsx}' --fix --c .eslintrc",
+    "lint": "eslint './src/**/*.{ts, tsx}' --c .eslintrc",
+    "test": "jest"
+  },
+  "author": "Ashish Padhy <100484401+Shurtu-gal@users.noreply.github.com>",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@amplication/code-gen-types": "^2.0.12",
+    "@amplication/code-gen-utils": "^0.0.6",
```

Comment:
@Shurtu-gal do you mind updating these dependencies to the latest? 
plugin-template should be up-to-date

Reply:
Sure.

Reply:
Done